soc/arm/nxp_imx/rt/soc_rt11xx.c

```diff
@@ -393,6 +401,52 @@ static ALWAYS_INLINE void clock_init(void)
 #endif
 }
 
+#if CONFIG_DISK_DRIVER_SDMMC
+
+/* Usdhc driver needs to re-configure pinmux
+ * Pinmux depends on board design.
+ * From the perspective of Usdhc driver,
+ * it can't access board specific function.
+ * So SoC provides this for board to register
+ * its usdhc pinmux and for usdhc to access
+ * pinmux.
+ */
```

Comment:
Is there an opportunity here to express this via a board level DTS instead of a board level function?

Reply:
In theory I think this could be moved to a board level DTS, but I believe that would require a pinmux driver, since as it stands we only directly use the IOMUXC calls in board level `pinmux.c` files. We could pull the DAT3 pin specifically into the DTS, and use the IOMUXC functions in the usdhc driver, but that seems like we are reimplementing part of the pinmux functionality